calcrule_third_party_payment/signals.py

```diff
@@ -6,6 +7,7 @@
 def bind_service_signals():
     bind_service_signal(
         'signal_after_claim_batch_module_process_batch_service',
-        ThirdPartyPaymentCalculationRule.convert_batch,
+        ThirdPartyPaymentCalculationRule.convert_batch_old,
```

Comment:
why did you change it to old ?

Reply:
because this was the old approach. This signal is not used in that approach (from develop branch in claim batch etc)

Reply:
Anyway this file is commented so as to return to previous solution in case of issues. But I can remove this if you prefer.

Reply:
Moreover I changed this 'convert batch' method because of using this 'work_data' from new batch run instead of taking kwargs of batch run execution like: audit_user_id, location_id, period, year

Reply:
@delcroip I removed old code. Now I think should be more readable.